docs/StardustDocs/topics/gradleReference.md

```diff
@@ -87,10 +104,61 @@ dataframes {
     }
 }
 ```
+## Schema definition for SQL table or result of an SQL query
+
+To generate schema for existing SQL table, 
+you need to define a few parameters to establish JDBC connection:
+URL (passing to `data` field), username and password.
+
+Also, the `tableName` parameter should be specified.
+
+```kotlin
+dataframes {
+    schema {
+        data = "jdbc:mariadb://localhost:3306/imdb"
```

Comment:
Can we add a link to this database?

Reply:
I want to release example with this database and will update the links